launcher/image/test/test_ssh_manual.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+set -euxo pipefail
+
+print_usage() {
+    echo "usage: test_ssh_manual.sh [-i imageName] [-p imageProject]"
+    echo "  -i <imageName>: which image name to use for the VM"
+    echo "  -p <imageProject>: which image project to use for the VM"
+    exit 1
+}
+
+run_ssh_test() {
+    BUILD_ID=$(date +%s)
+    HOME_DIR=$(echo ~)
+    VM_NAME="cs-ssh-test-$BUILD_ID"
+    WORKLOAD_IMAGE='us-west1-docker.pkg.dev/confidential-space-images-dev/cs-integ-test-images/basic-test:latest'
+    ZONE="us-central1-a"
+
+    ACCOUNT_NAME=$(gcloud config list account --format "value(core.account)" | tr @. _)
+    PROJECT_NAME=$(gcloud config get-value project)
+
+    # Create a new VM
+    source create_vm.sh -n $VM_NAME -i $IMAGE_NAME -p $IMAGE_PROJECT -m tee-image-reference=$WORKLOAD_IMAGE,tee-container-log-redirect=true,tee-cmd=["newCmd"],tee-env-ALLOWED_OVERRIDE=overridden,enable-osconfig=TRUE -z $ZONE
```

Comment:
I think `tee-cmd=["newCmd"],tee-env-ALLOWED_OVERRIDE=overridden,enable-osconfig=TRUE` is not needed in this test?

Reply:
I believe the first two options can be removed (and can do so following this) but OS Config is needed for OS Login as part of the BeyondCorp-style SSH requirements (see below comment).

Reply:
Yes we can remove the first two.
Also I think the osconfig is disabled on the hardened image, so is this this only works on debug image?

Reply:
That's correct. Though we want to keep the reference to OS Config here so we can ensure it is disabled for the hardened image even when set to be enabled. And it should work as regular for debug.